CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ This means that druid no longer requires cairo on macOS and uses Core Graphics i
 - Built-in open and save menu items now show the correct label and submit the right commands. ([#930] by [@finnerale])
 - Wheel events now properly update hot state. ([#951] by [@xStrom])
 - X11: Support mouse scrolling. ([#961] by [@jneem])
+- Add `Button::dynamic` constructor. ([#963] by [@totsteps])
```

Comment:
Additions should go into the `### Added` category.
Then you can just add
```
- `Button::dynamic` constructor. ([#963] by [@totsteps])
```
there.